web3swift/src/Contract/Statically Typed/EthereumClient+Static.swift

```diff
@@ -26,22 +26,41 @@ public extension ABIFunction {
 
     }
 
-    func call<T: ABIResponse>(withClient client: EthereumClientProtocol, responseType: T.Type, block: EthereumBlock = .Latest, completion: @escaping((EthereumClientError?, T?) -> Void)) {
+    func call<T: ABIResponse>(
+        withClient client: EthereumClientProtocol,
+        responseType: T.Type,
+        block: EthereumBlock = .Latest,
+        failOnExecutionError: Bool = true,
```

Comment:
Caller an still use previous behaviour (i.e. don't care if method exists, just default to 0/0x if it doesn't) for simplified logic